tests/cmd/e2e/main.go

```diff
@@ -55,6 +55,11 @@ func main() {
 	cli, kubeCli := client.NewCliOrDie()
 	ocfg := newOperatorConfig()
 
+	err = cleanAndInstallCRDs([]string{fmt.Sprintf("%s/e2e/crd.yaml", cfg.ManifestDir)})
```

Comment:
Use `kubectl apply -f crd.yaml` is more simpler.

Reply:
code is flexible

Reply:
I'll update it to use `kubectl`.

Reply:
done